### PR TITLE
fix: Remove unwanted arguments passed to HTTP methods

### PR DESCRIPTION
**Pull request by Nanrech**
## About

EDIT: This PR's purpose is now refactoring all edit/modify helper methods so that only the provided arguments are sent in the http payload

## Checklist

- [x] The ``pre-commit`` code linter has been run over all edited files to ensure the code is linted.
- [x] I've ensured the change(s) work on `3.8.6` and higher.

I've made this pull request: (check all that apply)
  - [ ] For the documentation
  - [ ] To add a new feature
  - [ ] As a general enhancement
  - [x] As a refactor of the library/the library's code
  - [x] To fix an existing bug
  - [ ] To resolve #ISSUENUMBER

This is:
  - [ ] A breaking change
  ^ Not sure

### CODE REVIEW

File: interactions/api/models/channel.py [hunk 1]
```diff
@@ -848,6 +846,11 @@ async def modify(
             payload["auto_archive_duration"] = auto_archive_duration
         if locked is not MISSING:
             payload["locked"] = locked
+        if position is MISSING:
+            if self.position is not None:
+                payload["position"] = self.position
```
Comment (mAxYoLo01) [inline interactions/api/models/channel.py]:
```suggestion
            if self.position is not None:
                payload["position"] = self.position
```
From what I see, the payload expects to get only the parameters which **need** to be modified. Therefore, why pass the current position, if we're not interested in changing it? Same logic appears for all parameters btw.

Comment (Nanrech) [reply interactions/api/models/channel.py]: That's a really good point. Committed out the redundant part.

Comment (mAxYoLo01) [reply interactions/api/models/channel.py]: Can you do the same for all non-necessary parameters?

**Comment by Damego** [comment]
Closed due to inactive

**Pull request closed without merging**